sigle/src/icons/FacebookLogoIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import { IconProps } from '@radix-ui/react-icons/dist/types';
```

Comment:
I noticed there's an existing `FacebookLogo.tsx` (different logo to this one), is it currently needed/being used elsewhere or could we remove/replace it with this one?